rulesdoc.md

```diff
@@ -4261,6 +4275,10 @@ Your experience with spell scrolls enables you to create spell scrolls for spell
 
 When a creature attacks you, you may make a reaction attack against them. This attack costs 2 AP, and you may only use this on the same creature once per round.
 
+#### Mana Recovery
+*Requires 3 proficiency in Arcane, 7th Level*
+
+When casting _Barrier_, _Counter Spell_, or blocking spell damage with _Elemental Shell_ you regain 1 mana, or 2 mana if the spell is 6th level and above, regardless of the rolls outcome.
```

Comment:
Interesting idea.  I was just thinking about making spell battles more interesting and this helps to do that.

I was also thinking about the potential for feats like "Spell Breaker" and "Spell Empowerment" which would give certain advantages in making counter spell stronger or overcoming an enemies counter spell.  That way if a player is frustrated by having their spells rejected or want to specialize in countering other mages, they have the options to take feats later that can enhance how they like to play.

Reply:
I kinda want the D&D wizard spell schools as feats. Like a set of feats that improve a subset of spells. Something like increasing AoE spells by 1 tile or making shielding spells more powerful, etc.

Reply:
Love this idea.